### DISCUSSION TITLE
cloud juicefs on windows

### DISCUSSION THREAD

**Discussion by netrc** [juicedata/juicefs#2372] [Q&A] [answered]
Using juicefs cloud edition, I've got a Google storage bucket attached to Linux just fine.

$ juicefs mount xyz-jfsg1 /jfs    # access works fine

Under Windows11, the juicefs.exe command appears to work differently? Can't find any docs on this use case.
The juicefs.exe commands I've seen also use option for specifying the meta-data database; shouldn't it default to juicefs cloud?

**Comment by xiaogaozi** [comment, marked as answer]
The Windows client of JuiceFS Community Edition cannot connect to the JuiceFS Cloud Service, you need to use the Windows client dedicated to the cloud service.

BTW, GitHub Discussions mainly discuss issues related to JuiceFS Community Edition. If you have any questions during the use of JuiceFS Cloud Service, please contact us through Intercom.

**Comment by netrc** [comment]
I'm told juicefs on Windows does not support cloud service yet.
